docs/operating/accounts.md

```diff
@@ -207,7 +207,7 @@ Best practice is to use the dedicated script (wip: coming soon!) to automaticall
 
 :::
 
-> Link to Docker compose file wip and coming soon!
+> Link to Docker compose file (wip) and coming soon!
```

Comment:
@thespamer this is WIP -- can we provide now?

Reply:
@thespamer what news?

Reply:
@himaster

Reply:
@himaster maybe https://github.com/neonlabsorg/proxy-model.py/tree/develop/docker-compose ?

Reply:
Tough to find links in resolved conversations!! In future @himaster please do a PR to update the document directly so that the tech writer need not play a game of hide and seek!!

Reply:
added to doc